lib/stats.js

```diff
@@ -67,7 +67,7 @@ var stats = {
       var childrens = options.childrens ? stat.cutime + stat.cstime : 0
       var total = 0;
 
-      if(history.utime) {
+      if(typeof history.utime !== 'undefined') {
```

Comment:
Ok, thanks. What about:

```
total = stat.stime - (history.stime || 0) + stat.utime - (history.utime || 0) + childrens
```

Btw `var history = this.history[pid] ? this.history[pid] : {}` could be `var history = this.history[pid] ? this.history[pid] : {utime: 0, stime: 0}` to avoid the condition. WDYT?